AndelaOSP/shufflebox-frontend-react
# Add calendar in the side panel

#### What does this PR do?
- Adds a calendar view to the shuffle page
#### Description of Task to be completed?
- Create a calendar showing markers for secret Santa, brown bag and hangouts events
- Add color codes and description of the events
#### How should this be manually tested?
- Once on the home page click `Get Started` to get to the shuffle page
- The calendar should be on the side panel
- Click on the right and left chevron to move to the next and previous month respectively
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#151598521](https://www.pivotaltracker.com/story/show/151598521)
#### Screenshots (if appropriate)
<img width="1440" alt="Calendar" src="https://github.com/AndelaOSP/shufflebox-frontend-react/blob/ft-calendar-view/src/images/calendar.png?raw=true">

#### Questions:
None
